lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -111,28 +144,45 @@
 
         AccountSettingsFieldViews.AuthFieldView = FieldViews.LinkFieldView.extend({
```

Comment:
This should be added in the same way you have added 3 keys to AccountSettingsFieldViews at the top of the function.